docs/quickstart.asciidoc

```diff
@@ -9,46 +9,42 @@ endif::[]
 
 [partintro]
 --
-With Elastic Cloud on Kubernetes (ECK) you can extend the basic Kubernetes orchestration capabilities to easily deploy, secure, upgrade your Elasticsearch cluster, and much more.
+With Elastic Cloud on Kubernetes (ECK) you can extend the basic Kubernetes orchestration capabilities to easily deploy, secure, upgrade your {es} cluster, and much more.
 
-Eager to get started? This quick guide shows you how to:
+Eager to get started? This quickstart guide shows you how to:
 
 * <<{p}-deploy-eck,Deploy ECK in your Kubernetes cluster>>
-* <<{p}-deploy-elasticsearch,Deploy an Elasticsearch cluster>>
-* <<{p}-deploy-kibana,Deploy a Kibana instance>>
-* <<{p}-upgrade-deployment,Upgrade your deployment>>
-* <<{p}-persistent-storage,Use persistent storage>>
-* <<{p}-check-samples,Check out the samples>>
+* <<{p}-deploy-elasticsearch,Deploy an {es} cluster>>
+* <<{p}-deploy-kibana,Deploy a {kib} instance>>
+* <<{p}-update-deployment,Update your deployment>>
 
-**Supported versions**
-
-include::supported-versions.asciidoc[]
-
---
```

Comment:
You should not remove this line (27, `--`) as it closes the section, this is the reason build is failing.